internal/service/service_test.go

```diff
@@ -0,0 +1,103 @@
+package service_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/andreipimenov/golang-training-2021/internal/mock"
+	"github.com/andreipimenov/golang-training-2021/internal/model"
+	"github.com/andreipimenov/golang-training-2021/internal/service"
+)
+
+const (
+	ticker          = "AAPL"
+	invalidTicker   = "aar"
+	date            = "2021-07-26"
+	dateWithoutData = "2021-08-07"
+)
+
+type serviceTestSuite struct {
+	suite.Suite
+	service  *service.Service
+	repoMock *mock.Repository
+}
+
+func (suite *serviceTestSuite) SetupTest() {
+	repo := &mock.Repository{}
+	s := service.New(&zerolog.Logger{}, repo, "1")
```

Comment:
The same issue as here: https://github.com/andreipimenov/golang-training-2021/pull/82#discussion_r689499835
and here: https://github.com/andreipimenov/golang-training-2021/pull/79#discussion_r694665193 . Please cooperate with guys and solve it.